terratag_test.go

```diff
@@ -23,14 +26,19 @@ var args = []string{
 	programName,
 	"-tags={\"env0_environment_id\":\"40907eff-cf7c-419a-8694-e1c6bf1d1168\",\"env0_project_id\":\"43fd4ff1-8d37-4d9d-ac97-295bd850bf94\"}",
 }
-var rootDir = "test/fixture"
```

Comment:
fixture directory contains  terreform versions, and each version has a config file.
I deleted all the tests from this directories and instead added a config file. (see config.yaml for an example).